### ISSUE TITLE
Is there a way to interact with Gazebo sim?

### ISSUE THREAD

**Issue by fgadaleta** [openrr/openrr#729]

**Comment by OTL** [comment]
You can find some examples of Gazebo.

https://github.com/openrr/openrr/tree/main/openrr-apps

**Comment by fgadaleta** [comment]
Yes and no. I don't want to depend on ROS. In fact, in my particular case, using ROS is not an option.

**Comment by taiki-e** [comment]
I'm not sure if we have the bandwidth to investigate and implement it on our end, but it seems okay to provide support for gazebo's ros-independent API if it has one.
